controllers/index.js

```diff
@@ -26,17 +27,22 @@ const getReviews = (req, res, next) => {
 }
 
 const getReviewById = (req, res, next) => {
-    const id = req.params.review_id;
-    readReviewById(id).then((review) => {
+    readReviewById(req.params.review_id).then((review) => {
         res.status(200).send({ review: review });
     }).catch((err) => {
-        console.log(err);
         next(err);
     })
 }
 
```

Comment:
Here you will need to make a checkIfReviewExists function that allows you to differentiate between a valid review with no comments (200) and a non existent review (404). Both of these will return an empty array at this point. This function will make a query to the database, meaning you will need two queries in total. 

See these notes for assistance: 

https://notes.northcoders.com/courses/js-back-end/advanced-error-handling